swap/src/cli.rs

```diff
@@ -1,38 +1,74 @@
-use libp2p::core::Multiaddr;
+use libp2p::{core::Multiaddr, PeerId};
+use std::str::FromStr;
 use url::Url;
 use uuid::Uuid;
 
 #[derive(structopt::StructOpt, Debug)]
 #[structopt(name = "xmr-btc-swap", about = "Trustless XMR BTC swaps")]
 pub enum Options {
     Alice {
-        #[structopt(default_value = "http://127.0.0.1:8332", long = "bitcoind")]
+        #[structopt(
+            short = "b",
+            long = "bitcoind",
+            default_value = "http://127.0.0.1:8332"
+        )]
         bitcoind_url: Url,
 
-        #[structopt(default_value = "http://127.0.0.1:18083/json_rpc", long = "monerod")]
-        monerod_url: Url,
+        #[structopt(
+            short = "m",
+            long = "monero-wallet-rpc",
+            default_value = "http://127.0.0.1:18083/json_rpc"
```

Comment:
Does a default value make sense? From our discussion looks like not.

Reply:
Yes, it does. This is the wallet-rpc andpoint, that typically runs at `18083`, but there is no official default port defined.